postlite/src/customize/App.vue

```diff
@@ -0,0 +1,89 @@
+<template>
```

Comment:
postlite/src/customize directory should be deleted.
customize mode code has moved to /postlite-customize